src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Components/SortableHeaderCell/SortableHeaderCellViewModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Web;
+using Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Models;
+
+namespace Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Components.SortableHeaderCell;
+
+public class SortableHeaderCellViewModel
+{
+    public string DisplayName { get; set; }
+    public string Key { get; set; }
+    public SortDirection? SortDirection { get; set; }
+    public bool IsActive { get; set; }
```

Comment:
Isn't nullable sort direction enough?

Reply:
IsActive is needed to show glyphs only for active column.